src/pages/ClaimsPage/ClaimsPage.tsx

```diff
@@ -23,10 +23,12 @@ const currentClaims = (claims: QueryClaim[]) =>
       (finalClaims, curr) =>
         finalClaims.filter(
           c =>
-            c.distribution.vault.vault_address ==
+            c.distribution.vault.vault_address ===
               curr.distribution.vault.vault_address &&
             c.distribution.epoch.circle?.id ===
-              curr.distribution.epoch.circle?.id
+              curr.distribution.epoch.circle?.id &&
+            c.distribution.vault.simple_token_address ===
```

Comment:
this additional simple_token_address check isn't necessary; you can't have a case where vault_address is the same and either of the token addresses differ